tests/clims/api/endpoints/test_substances.py

```diff
@@ -1,57 +1,76 @@
 from __future__ import absolute_import
 
+import six
 import random
 import json
 
 from django.core.urlresolvers import reverse
 
 from sentry.testutils import APITestCase
 
-from tests.clims import testutils
 from rest_framework import status
 from clims.models import Substance
+from tests.fixtures.plugins.gemstones_inc.models import GemstoneSample
 
 
 class SubstancesTest(APITestCase):
+    def create_gemstone(self, *args, **kwargs):
+        return self.create_substance(GemstoneSample, *args, **kwargs)
+
     def test_get_substances(self):
-        substance = testutils.create_substance()
+        first = self.create_gemstone(color='red')
+        first.color = 'blue'
+        first.save()
 
-        url = reverse('clims-api-0-substances', args=(substance.organization.name,))
+        url = reverse('clims-api-0-substances', args=(first.organization.name,))
         self.login_as(self.user)
-
-        url = reverse('clims-api-0-substances', args=("lab",))
         response = self.client.get(url)
         len_before = len(response.data)
 
-        substance = testutils.create_substance()
+        second = self.create_gemstone()
         response = self.client.get(url)
 
         assert response.status_code == 200, response.content
+
+        # NOTE: This may not work if tests run async on the same DB
```

Comment:
Am I wrong in assuming that this works as expected now?

Reply:
I think my point here was that if we start running the tests async, the delta would possibly be more than 1. But if we use the django asynch features, the system creates one test database per process, so that should not be an issue. I'll remove the comment.